lib/ts/framework/awsLambda/framework.ts

```diff
@@ -45,7 +44,7 @@ export class AWSRequest extends BaseRequest {
         if (this.event.body === null || this.event.body === undefined) {
             return {};
         } else {
-            const parsedUrlEncodedFormData = parse(this.event.body);
+            const parsedUrlEncodedFormData = Object.fromEntries(new URLSearchParams(this.event.body).entries());
```

Comment:
If someone passes a malformed body how does that show up? Is the error message correct?

Reply:
Fixed it, it will throw a input error if that happens.